inc/Classes/Database.php

```diff
@@ -72,10 +77,27 @@ public function connect(): bool
         // If it is _not_ successful, an exception will be thrown.
     
         $this->setCharset($this->charset);
+        $this->setConnectionStatus(true);
 
         return true;
     }
 
+    /**
+     * Sets the connection status
+     */
+    private function setConnectionStatus(bool $status): void
```

Comment:
I don't see the added value of a private setter function especially given the simple code it contains. 
But fine for me as code is working and still better than what we had ;)